intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -0,0 +1,162 @@
+| A class that describes an ordered list                                                                    | List       |
+| A class that describes a vehicle driver working at a ride-share app, who has a ride history and a vehicle | Driver     |
+| A class that describes a music album sold in a digital music store, and all its data                      | Album      |
+
+### !end-callout
+
+### Creating Instances of Classes
+
+An **instance** is a concrete, literal manifestation of a class.
+
+A _class_ defines a data type _generically_, much like a template, or blueprint. It determines what that data type _can potentially_ hold as state or behavior.
+
+On the other hand, an _instance_ of a class is a _concrete thing_ of that data type, with concrete state and behavior. One instance is one specific thing that has literal state and literal behavior. Each instance allocates and uses memory, just like other pieces of data.
+
+In our code, after a class is defined, we can create one instance, 50 instances, or thousands of instances. All of these instances will be unique from each other, because they can all hold different attributes, and occupy different places in memory.
+
+### !callout-info
+
+## Examples: Instances of Classes
+
+Observe these examples of classes that already exist, or classes that could exist.
+
+| Class  | One Instance of the Class                      | Another Instance of the Class                     |
+| ------ | ---------------------------------------------- | ------------------------------------------------- |
+| String | `"Hello World! I am an instance of a String!"` | `"The quick brown fox jumped over the lazy dog."` |
+| List   | `["red", "orange", "yellow"]`                  | `["June", "July", "August"]`                      |
+| Driver | A driver with a driver ID of `DR0004`          | A driver with a driver ID of `DR0005`             |
+| Album  | Dirty Computer (2018)                          | Purple Rain (1984)                                |
+
+### !end-callout
+
+#### Synonyms for Creating Instances
+
+Creating instances is something we'll do a lot, almost as much as making variables or using loops. There are a lot of ways to say that we're creating an instance.
+
+We might say...
+
+- I'm making an instance of an `Album`. This instance represents one album, named "Purple Rain."
+- I'm creating an instance of `Album`. This instance represents one album, "Purple Rain."
+- I'm instantiating the `Album` class. Now I have an instance, which is "Purple Rain."
+- I'm making an `Album` object. The `Album` object is the "Purple Rain" album.
+- The "Purple Rain" instance of `Album` is very different from the "Dirty Computer" instance of `Album`.
+- "Purple Rain" and "Dirty Computer" are both instances of `Album`.
+
+### State and Behavior
+
+When we start defining our own classes, we will define their generic **state** and their generic **behavior.**
```

Comment:
```suggestion
When we start defining our own classes, we will specify their **state** and their generic **behavior.**
```

For some reason I feel pretty strongly about keeping future tense for this intro-intro-conceptual lesson

(the "some reason" is something about acknowledging that the journey is about to start and inviting them, and I have this whole pedagogical thing about inviting students and giving them permission to start doing something they've never done before by saying "we will do this" rather than "when we do this thing and it kinda sounds like you might already be doing it but you probably aren't so it sounds like I'm addressing people who are already doing it")

(Maybe one day I'll write that in a more coherent form. Sorry)

Reply:
Ah.  Fair point.  It was intended to be more forward looking than I had read it.